.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+node_js:
+  - "4.2"
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm install -g selenium-standalone
+  - selenium-standalone install
+  - selenium-standalone start > /dev/null 2>&1 &
```

Comment:
I have opened up #21 to patch this properly. The Travis commands should now be corrected to 1:1 with what we have in #21's docs:

```yml
install:
    # Install node modules
    - npm install

    # Install Selenium server
    - npm run webdriver-manager-update

before_script:
# ...
    - sleep 3 # give xvfb some time to start

    # Start our Selenium server
    - npm run webdriver-manager-start &> /dev/null &
    - sleep 3 # Give selenium some time to start
```